docs/archive.md

```diff
@@ -0,0 +1,26 @@
+# Data Archiving
+
+When the crawls are complete, the data is archived to several places:
+
+- Internet Archive (sql)
+- HTTP Archive (sql, csv)
+- Google Storage (har)
+- BigQuery (har)
+
```

Comment:
@Themanwithoutaplan we are moving away from MySQL. Those dumps are a byproduct of the current site, which relies on a custom rollup+processing pipeline to aggregate results... which doesn't lend itself to reprocessing and very hard to update. 

We will continue providing the raw dataset + BigQuery aggregates that should closely match what you see today (and can query via API or export yourself), plus more.. If you haven't already definitely take a look at the BigQuery route.